dla_cnn/desi/preprocess.py

```diff
@@ -92,3 +92,68 @@ def label_sightline(sightline, kernel, REST_RANGE, pos_sample_kernel_percent=0.3
     # classification is 1 / 0 / -1 for DLA/nonDLA/border
     # offsets_array is offset
     return classification, offsets_array, column_density
+
+def rebin(sightline, v = 20000):
+    """
+    Resample and rebin the input Sightline object's data to a constant dlambda/lambda dispersion.
+
+    Parameters
+    ----------
+    sightline: dla_cnn.data_model.Sightline
+    v: float
+    
+    Returns
+    -------
+    sightline: dla_cnn.data_model.Sightline
+    """
+    c = 2.9979246e8
+    dlambda = np.log(1+v/c)
+    wavelength = np.exp(np.log(10)*sightline.loglam)
+    central_wavelength = [item.central_wavelength for item in sightline.dlas]
+    max_wavelength = wavelength[-1]/min(central_wavelength)
```

Comment:
I don't see understand this line.  I would just use
`max_wavelength = wavelength[-1]`

Reply:
Same confusion for `min_wavelength`

Reply:
Emmm, I referenced to the linetools. https://github.com/linetools/linetools/blob/31355147d3d8d874b0a61a5d9a528b1e7e3e6909/linetools/spectra/utils.py#L358
Maybe I should make a little change.

Reply:
Also, I am curious about why we define wvmax and wvmin like that in linetools.

Reply:
Ah, that method you reference is for rebinning in the rest frame.
We may wish to put the spectrum in the rest-frame, but if we do
it would be the quasar restframe, not the DLAs. 

But let's ignore restframe for now.

Reply:
Ok, thanks.